docworks-dts/test/multiple-files/__snapshots__/main.test.js.snap

```diff
@@ -804,2960 +804,2960 @@ namespace $w {
 
 }
 /**
- * The wix-users module contains functionality for working with your
- *  site's users from client-side code.
- * 	[Read more..](https://fake-corvid-api/wix-users.html#)
+ * The wix-crm module contains functionality for working with
```

Comment:
tagid, did we understood why those snapshots changed

Reply:
from a quick look i think its related to array-flatten package in docworks-repo which sorts the services files

Reply:
(and flatten them also... )